README.md

```diff
@@ -87,84 +105,104 @@ class ConnectionProvider(Provider):
+   For more details, see [integrations doc.](https://dishka.readthedocs.io/en/latest/integrations/index.html)
 
 ```python
 from dishka.integrations.fastapi import (
     FromDishka, inject, setup_dishka,
 )
 
+
 @router.get("/")
 @inject
 async def index(service: FromDishka[Service]) -> str:
     ...
 
+
 ...
 setup_dishka(container, app)
 ```
 
 ### Concepts
 
-**Dependency** is what you need for some part of your code to work. They are just object which you do not create in place and probably want to replace some day. At least for tests.
-Some of them can live while you application is running, others are destroyed and created on each request. Dependencies can depend on other objects, which are their dependencies.
+**Dependency** is what you need for some parts of your code to work.
+Dependencies are simply objects you don't create directly in place and might want to replace someday, at least for
+testing purposes.
+Some of them live throughout your application runtime, while others are created and destroyed with each request.
+Dependencies can also rely on other objects, which then become their dependencies.
 
-**Scope** is a lifespan of a dependency. Standard scopes are (without skipped ones):
+**Scope** is the lifespan of a dependency. The default scopes are (with some skipped):
 
-  `APP` -> `REQUEST` -> `ACTION` -> `STEP`.
+`APP` -> `REQUEST` -> `ACTION` -> `STEP`.
 
-You decide when to enter and exit them, but it is done one by one. You set a scope for your dependency when you configure how to create it. If the same dependency is requested multiple time within one scope without leaving it, then by default the same instance is returned.
+You decide when to enter and exit each scope, but this happens one by one.
+You assign a scope to each dependency when configuring its creation.
+By default, if the same dependency is requested multiple times within a single scope without leaving it, the same
+instance is returned.
 
-If you are developing web application, you would enter `APP` scope on startup, and you would `REQUEST` scope in each HTTP-request.
+For a web application, enter `APP` scope on startup and `REQUEST` scope for each HTTP request.
 
-You can provide your own Scopes class if you are not satisfied with standard flow.
+If the default scope flow doesn't fit your needs, you can create a custom scope by defining your own `Scope` class.
 
-**Container** is what you use to get your dependency. You just call `.get(SomeType)` and it finds a way to get you an instance of that type. It does not create things itself, but manages their lifecycle and caches. It delegates objects creation to providers which are passed during creation.
+**Container** is what you use to get your dependencies.
+You simply call `.get(SomeType)`, and it finds a way to provide you with an instance of that type.
+The container itself doesn't create objects but manages their lifecycle and caches.
+It delegates object creation to providers added during setup.
```

Comment:
I cannot say `added`, there is nothing to add to. They are just passed and used together

Reply:
Ok